jetty-start/src/test/java/org/eclipse/jetty/start/fileinits/MavenLocalRepoFileInitializerTest.java

```diff
@@ -172,28 +177,90 @@ public void testGetCoordinateTestMavenBaseUri()
     }
 
     @Test
-    public void testDownloaddefaultrepo()
+    public void testDownloadDefaultRepo()
         throws Exception
     {
         MavenLocalRepoFileInitializer repo =
             new MavenLocalRepoFileInitializer(baseHome, null, false);
-        String ref = "maven://org.eclipse.jetty/jetty-http/9.4.10.v20180503/jar/tests";
+        String ref = "maven://org.eclipse.jetty/jetty-http/9.4.31.v20200723/jar/tests";
         Coordinates coords = repo.getCoordinates(URI.create(ref));
         assertThat("Coordinates", coords, notNullValue());
 
         assertThat("coords.groupId", coords.groupId, is("org.eclipse.jetty"));
         assertThat("coords.artifactId", coords.artifactId, is("jetty-http"));
-        assertThat("coords.version", coords.version, is("9.4.10.v20180503"));
+        assertThat("coords.version", coords.version, is("9.4.31.v20200723"));
         assertThat("coords.type", coords.type, is("jar"));
         assertThat("coords.classifier", coords.classifier, is("tests"));
 
         assertThat("coords.toCentralURI", coords.toCentralURI().toASCIIString(),
-            is("https://repo1.maven.org/maven2/org/eclipse/jetty/jetty-http/9.4.10.v20180503/jetty-http-9.4.10.v20180503-tests.jar"));
+            is("https://repo1.maven.org/maven2/org/eclipse/jetty/jetty-http/9.4.31.v20200723/jetty-http-9.4.31.v20200723-tests.jar"));
 
-        Path destination = Paths.get(System.getProperty("java.io.tmpdir"), "jetty-http-9.4.10.v20180503-tests.jar");
+        Path destination = testdir.getEmptyPathDir().resolve("jetty-http-9.4.31.v20200723-tests.jar");
         Files.deleteIfExists(destination);
         repo.download(coords.toCentralURI(), destination);
         assertThat(Files.exists(destination), is(true));
-        assertThat(destination.toFile().length(), is(962621L));
+        assertThat(destination.toFile().length(), is(986193L));
+    }
+
+    @Test
+    public void testDownloadSnapshotRepo()
+        throws Exception
+    {
+        Path snapshotLocalRepoDir = testdir.getPath().resolve("snapshot-repo");
+        FS.ensureEmpty(snapshotLocalRepoDir);
+
+        MavenLocalRepoFileInitializer repo =
+            new MavenLocalRepoFileInitializer(baseHome, snapshotLocalRepoDir, false, "https://oss.sonatype.org/content/repositories/jetty-snapshots/");
```

Comment:
I don't like much the idea about having test depending on downloading resources from https://oss.sonatype.org.
@joakime not urgent but I can fix that (next week when back from holidays)

Reply:
We could setup a small remote repo in `src/test/resources` (a 1 artifact repo) for testing purposes.

Reply:
I'll work on this today.